src/Microsoft.DotNet.VersionTools/BuildInfo.cs

```diff
@@ -180,6 +189,23 @@ public static string RawBuildInfoBaseUrl(string rawRepoUrl, string gitRef, strin
             return packages;
         }
 
+        private static async Task<HttpResponseMessage> GetBuildInfoFileAsync(
+            HttpClient client,
+            string buildInfoName,
+            string url)
+        {
+            HttpResponseMessage response = await client.GetAsync(url);
+            if (!response.IsSuccessStatusCode)
+            {
+                Trace.TraceError(
+                    $"Failure response code while fetching BuildInfo: name '{buildInfoName}', " +
+                    $"url: '{url}'. Ensure the repository is correct and the file exists at " +
```

Comment:
nit - why a colon after url but not name?

Reply:
Whoops, just bad proofreading. Fixing.